Fix installation of blowfish and DES headers on case-insensitive HFS+.

http://trac.macports.org/changeset/14310

--- crypto/bf/Makefile.orig	2005-09-28 16:51:37.000000000 -0700
+++ crypto/bf/Makefile	2005-09-28 16:52:25.000000000 -0700
@@ -2,6 +2,9 @@
 # OpenSSL/crypto/blowfish/Makefile
 #
 
+# Case-insensitive HFS+
+.PHONY: install
+
 DIR=	bf
 TOP=	../..
 CC=	cc
--- crypto/des/Makefile.orig	2005-09-28 16:51:42.000000000 -0700
+++ crypto/des/Makefile	2005-09-28 16:52:15.000000000 -0700
@@ -2,6 +2,9 @@
 # OpenSSL/crypto/des/Makefile
 #
 
+# Case-insensitive HFS+
+.PHONY: install
+
 DIR=	des
 TOP=	../..
 CC=	cc
